src/praisonai/tests/unit/test_approval_basic.py

```
@@ -0,0 +1,265 @@
+import sys
+import os
+import asyncio
+
+# Add the praisonai-agents module to path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..', '..', 'praisonai-agents'))
+
+def test_imports():
+    """Test that all the new approval imports work correctly."""
+    try:
+        from praisonaiagents.approval import (
+            require_approval,
```

Ruff (0.11.9)

`praisonaiagents.approval.require_approval` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)